src/app/(proper_react)/redesign/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/welcome-to-premium/WelcomeToPlus.stories.tsx

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import type { Meta, StoryObj } from "@storybook/react";
+import { OnerepScanRow } from "knex/types/tables";
+import { WelcomeToPlusView } from "./WelcomeToPlusView";
+import {
+  createRandomBreach,
+  createRandomScanResult,
+  createUserWithPremiumSubscription,
+} from "../../../../../../../../../../apiMocks/mockData";
+import { Shell } from "../../../../../../../Shell";
+import { getOneL10nSync } from "../../../../../../../../../functions/server/mockL10n";
+import { LatestOnerepScanData } from "../../../../../../../../../../db/tables/onerep_scans";
+
+const mockedScan: OnerepScanRow = {
+  created_at: new Date(1998, 2, 31),
+  updated_at: new Date(1998, 2, 31),
+  id: 0,
+  onerep_profile_id: 0,
+  onerep_scan_id: 0,
+  onerep_scan_reason: "initial",
+  onerep_scan_status: "finished",
+};
+
+function getMockedScanData(brokerScanCount: number): LatestOnerepScanData {
+  return {
+    scan: mockedScan,
+    results: [...Array(brokerScanCount)].map(() =>
+      createRandomScanResult({ status: "new", manually_resolved: false }),
+    ),
+  };
+}
+
+const mockedBreaches = [...Array(5)].map(() => createRandomBreach());
+
+const user = createUserWithPremiumSubscription();
+
+const mockedSession = {
+  expires: new Date().toISOString(),
+  user: user,
+};
+
+const WelcomeToPlusViewWrapper = (props: { brokerScanCount: number }) => {
+  return (
+    <>
```

Comment:
nit: Not sure this needs the wrapper `<>` since there is only one child.

Reply:
Removed in [b2c8f63](https://github.com/mozilla/blurts-server/pull/3997/commits/b2c8f63f0ae85cf10f7e3c4bc3507522fbb47da8).